### ISSUE TITLE
3 additional types of shoulder pain

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10991]
**Preferred term label:**
Posterior shoulder pain
Superior shoulder pain
Lateral shoulder pain

**Synonyms**

**Definition (free text, please give PubMed ID)**
Posterior: Pain localized to the posterior aspect of the shoulder
Superior: Pain localized to the superior or proximal aspect of the shoulder over the acromion, distal clavicle, and/or acromioclavicular (AC) joint
Lateral: Pain localized to the lateral aspect of the shoulder

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/browse/term/HP:0030834

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Posterior: Shoulder internal impingement syndrome, Suprascapular neuropathy, Labrum tear, shoulder, Shoulder dislocation
Superior: Acromioclavicular (AC) joint injury, Acromioclavicular (AC) joint osteoarthritis (OA), Distal clavicle osteolysis (DCO)
Lateral: Proximal humerus fracture, Proximal humeral epiphysiolysis, Adhesive capsulitis, Suprascapular neuropathy

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001221
- label: Posterior shoulder pain
- parents: [Shoulder pain (HP:0030834)](https://hpo.jax.org/app/browse/term/HP:0030834)
- pmid: 
- def: An unpleasant sensation characterized by physical discomfort (such as pricking, throbbing, or aching) localized to the posterior (back) part of the shoulder.
- comment: Posterior shoulder pain may be observed in conditions such as shoulder internal impingement syndrome, suprascapular neuropathy, labrum tear, and shoulder dislocation.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10991
- synonyms:

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001222
- label: Superior shoulder pain
- parents: [Shoulder pain (HP:0030834)](https://hpo.jax.org/app/browse/term/HP:0030834)
- pmid: 
- def: An unpleasant sensation characterized by physical discomfort (such as pricking, throbbing, or aching) localized to the superior or proximal aspect of the shoulder over the acromion, distal clavicle, and/or acromioclavicular (AC) joint.
- comment: Superior shoulder pain may be observed in conditions such as acromioclavicular joint injury, acromioclavicular joint osteoarthritis, and distal clavicle osteolysis.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10991
- synonyms:

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001223
- label: Lateral shoulder pain
- parents: [Shoulder pain (HP:0030834)](https://hpo.jax.org/app/browse/term/HP:0030834)
- pmid: 
- def: An unpleasant sensation characterized by physical discomfort (such as pricking, throbbing, or aching) localized to the localized aspect (on the side) of the shoulder.
- comment: Lateral shoulder pain may be observed in conditions such as proximal humerus fracture, proximal humeral epiphysiolysis, adhesive capsulitis, and suprascapular neuropathy.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10991
- synonyms:
